clang/tools/3c/utils/port_tools/generate_ccommands.py

```diff
@@ -18,7 +18,7 @@
 VSCODE_SETTINGS_JSON = os.path.realpath("settings.json")
 
 # to separate multiple commands in a line
-CMD_SEP = " &"
+CMD_SEP = " &&"
```

Comment:
Where was this used? I'm surprised that this hasn't caused an issue yet.

Reply:
Only in the "individual commands file", which has one `3c` command per translation unit.  AFAIK, we currently don't use that file for anything.  Is it better if I revert the change from this PR and we do a broader cleanup of `clang/tools/3c/utils` (removing the code we don't plan to use and fixing the rest) later?

Reply:
I guess we are keeping this change in this PR.